script/release.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# Constants. Enter relevant repo information here.
+UPSTREAM_REPO="kubernetes-incubator"
+CLI="kompose"
+
+usage() {
+  echo "This will prepare $CLI for release!"
+  echo ""
+  echo "Requirements:"
+  echo " git"
+  echo " gpg - with a valid GPG key already generated"
+  echo " tar - you should already have this on your system"
+  echo " hub"
+  echo " github-release"
+  echo " GITHUB_TOKEN in your env variable"
+  echo " "
+  echo "Not only that, but you must have permission for:"
+  echo " Tagging releases within Github"
+  echo ""
+}
+
+requirements() {
+  if [ ! -f /usr/bin/git ] && [ ! -f /usr/local/bin/git ]; then
+    echo "No git. What's wrong with you?"
```

Comment:
I would remove those "what's wrong with you". I understand it is funny, but some folks may not like it.